packages/ds-healthcare-gov/src/components/Accordion/Accordion.stories.jsx

```diff
@@ -12,13 +12,7 @@ export default {
     openItems: [0],
   },
   subcomponents: { AccordionItem },
-  decorators: [
-    (Story) => (
-      <div data-theme="healthcare">
-        <Story />
-      </div>
-    ),
-  ],
+  decorators: [(Story) => <div data-theme="healthcare">{Story()}</div>],
```

Comment:
I don't think the `data-theme` stuff is necessary here, I removed it locally from `stars` and it came up fine. Since users of the react components won't be using the data-theme attribute I think we can just return the story without wrapper.